### DISCUSSION TITLE
Flashing HFEASY and going back to original firmware

### DISCUSSION THREAD

**Discussion by shinhoshi57** [ljalves/hfeasy#33] [General]
Hi,
I've been playing a bit with HFEASY and generating a few binaries which correctly deployed to an LPT230.

The thing is that when I wanted to go back to the original image fw by HF, the upgrade module refuses to upgrade it.

Deploying a custom fw made by the official SDK results in the same. 

Any firmware created with HFEASY installs no problem.

Any idea?

Bests,

E

**Comment by ljalves** [comment]
Usually there are 2 types of images created by the sdk. One for serial upgrade and another for web flasher.
Check if you are using the correct one.
